bin/wabur

```diff
@@ -129,15 +129,24 @@ options = {
 
 config = WAB::Impl::Configuration.new(usage, options)
 mode = config[:mode]
+# TBD: sanitize path to ensure ancestor-level directories cannot be written to.
+dir = File.expand_path(config[:base] || '.')
 case mode
 when 'new'
-  dir = File.expand_path(config[:base] || '.')
-  abort(WAB::Impl.format_error("#{dir} already exists. Refusing to over-write. Use init instead.")) if Dir.exist?(dir)
+  if Dir.exist?(dir)
+    if Dir.pwd == dir
+      msg = "Refusing to over-write current directory. Either try again with a path to `--base` or use `wabur init` instead."
+    else
+      msg = "#{dir} already exists. Refusing to over-write. Use `wabur init` with '--base #{config[:base]}' instead."
+    end
+    abort(WAB::Impl.format_error(msg))
+  end
   FileUtils.mkdir_p(dir)
   WAB::Impl::Init.setup(dir, config)
 when 'init'
-  dir = File.expand_path('.')
-  WAB::Impl::Init.setup(dir, config)
+  Dir.chdir(dir) do
```

Comment:
Why do you think this is needed. Init does not use relative paths.

Reply:
so that the user need not manually `cd` into their existing subdirectory and *then run* `wabur init`

Reply:
The code is already running init. The user does not need to run it again. Besides, one the program exits the change in directory is no longer in effect.

Reply:
hmm.. did not consider that angle.. the user will have to `cd` into their `--base` to run the App eitherways.. sad..

Reply:
changing directories is a pretty trivial operation. If the user is having trouble with that they will have serious problems running any software.

Reply:
The main intention for this PR was to **inform the user to change directory to their desired `--base`** and then run `wabur init`.. but it escalated into this..